lib/kubernetes-deploy/kubernetes_resource.rb

```diff
@@ -192,5 +185,51 @@ def pretty_status
     def kubectl
       @kubectl ||= Kubectl.new(namespace: @namespace, context: @context, logger: @logger, log_failure_by_default: false)
     end
+
+    class Event
+      JSONPATH_FIELDS =
+        "{.involvedObject.kind}\t{.involvedObject.name}\t{.count}\t{.lastTimestamp}\t{.reason}\t{.message}"
+      JSONPATH_SEPARATOR = "ENDEVENT\nBEGINEVENT"
+
+      def self.jsonpath_for(name)
+        %({range .items[?(@.involvedObject.name=="#{name}")]}#{JSONPATH_FIELDS}#{JSONPATH_SEPARATOR}{end})
```

Comment:
Looks like you can emit \n with jsonpath if you make it part of jsonpath string literal, i.e. instead of just `\n` make it `{"\n"}`